lib/pure/asyncdispatch.nim

```diff
@@ -953,6 +962,117 @@ when defined(windows) or defined(nimdoc):
     ## Unregisters ``fd``.
     getGlobalDispatcher().handles.excl(fd)
 
+  {.push stackTrace:off.}
+  proc waitableCallback(param: pointer,
+                        TimerOrWaitFired: WINBOOL): void {.stdcall.} =
+    var p = cast[PPCD](param)
+    discard postQueuedCompletionStatus(p.ioPort, Dword(TimerOrWaitFired),
```

Comment:
Of course it can fail, but technically this code would run in other thread, so its why here `{.push stackTrace:off.} to avoid any nim's specific `under dress` code... But if i raise exception in this function, something can go wrong and program can fall into more serious error.